### DISCUSSION TITLE
Admin Features/Requests

### DISCUSSION THREAD

**Discussion by pevey** [medusajs/medusa#4992] [Feature Requests]
I know a lot of work is being done on the admin, so I am dropping here a few things I've noticed, a wishlist of sorts.  I am tempted to work on some of these things (I actually did submit a PR for one a long while back), but I feel like there are big changes going on right now with the admin, and a PR would from someone else might be tough to manage with the rapid changes:

1) The sidebar needs to be collapsible.  This is really all that is needed to enable a kick*ss POS plugin for Medusa that would look great on phones and tablets.

2) The admin is still installing 2 separate versions of react.  You can see this by doing yarn list react  React-nestable is a dependency that has not been kept up to date, and using it requires an old version.

3) The admin is pulling in an old version of fetch that has security vulns.  This is not immediately a concern for me since my admin is firewalled, but it doesn't look good when people install the project for the first time.

4) ~~Would it be super difficult to make all the strings variables, to enable localization of the admin?~~ Done!

**Comment by olivermrbl** [comment]
Thanks for sharing this, Lacey! I'll forward these to our team.

As for 4., we are aiming to merge https://github.com/medusajs/medusa/pull/4962 this week, enabling localization of admin 👍
